test/parallel/test-cluster-disconnect-race.js

```diff
@@ -7,6 +7,12 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 const cluster = require('cluster');
+
+if (common.isWindows) {
+  console.log('1..0 # Skipped: This test is disabled on windows.');
```

Comment:
Nit: `Windows` rather than `windows`

Nit: `is disabled on Windows` makes it sound like we just gave up and hope to enable it one day. I'd rather convey a message that it just doesn't apply to Windows. Maybe: `Skipped: This test does not apply to Windows.`